### ISSUE TITLE
1.1.8 not updating the Windows environment variables

### ISSUE THREAD

**Issue by ArloL** [graalvm/setup-graalvm#88]
I am using the action [like this](https://github.com/ArloL/chorito/blob/main/.github/workflows/main.yaml#L142) on windows-latest:

```
- uses: graalvm/setup-graalvm@v1.1.7
  with:
    version: 22.2.0
    java-version: 17
    components: 'native-image'
    github-token: ${{ secrets.GITHUB_TOKEN }}
    cache: 'maven'
```

and now with 1.1.8 my windows setup stopped working.

My guess is that something is tripping up the required update to the environment variables which is still required with these older version graal versions I am sadly still stuck on.

The 1.1.7 log contains
```
Updating Windows environment...
```

while 1.1.8 does not.

Please let me know if I can provide more info.

PS Thanks for all the hard work maintaining open source software 🙏🏼

**Comment by fniephaus** [comment]
Thanks for the report, @ArloL!

> and now with 1.1.8 my windows setup stopped working.

Could you provide us with a link to a failing build so that we can take a closer look?

**Comment by fniephaus** [comment]
It seems this could be related to https://github.com/graalvm/setup-graalvm/commit/0fee1abbca81264b6090dedc29ae89311bb9185b. I will try and come up with a fix tomorrow.

**Comment by fniephaus** [comment]
I just pushed a fix for this to dev, you can try it out via `uses: graalvm/setup-graalvm@dev`.

**Comment by ArloL** [comment]
Thanks for the quick reply ☺️

This is the build in question: 
https://github.com/ArloL/chorito/actions/runs/8016511422/job/21923586863?pr=320

I would love to use the latest graalvm and java 21 but I am sadly hitting this issue https://github.com/oracle/graal/issues/6793

Sadly @dev also does not update the windows environment: https://github.com/ArloL/chorito/actions/runs/8244996369/job/22548159025

**Comment by bhavikp19** [comment]
@fniephaus I can confirm that my windows build was also failing but not it is working if I use `@dev`

**Comment by fniephaus** [comment]
The fix is merged and available via `v1.1.8.2` and `v1`.

**Comment by ArloL** [comment]
Thanks, @fniephaus. I am not sure but I think dependabot will not update to 1.1.8.2. I tried recreating the PR:
https://github.com/ArloL/chorito/pull/320#issuecomment-1992097884 but the build still just used 1.1.8. I will wait and see if dependabot will open a new PR.

**Comment by fniephaus** [comment]
Why will dependabot not update to that version? Do we need to bump the patch version?

**Comment by fniephaus** [comment]
I just bumped the version to `v1.1.9`. Let me know if dependabot picks it up :)

**Comment by ArloL** [comment]
Thanks. Now dependabot closed the 1.1.8 issue and opened another PR for 1.1.9: https://github.com/ArloL/chorito/pull/333

I am not sure why exactly that is necessary to be honest. My guess is that they only check for x.y.z version schemes and not x.y.z.a

**Comment by fniephaus** [comment]
Thanks for confirming! We'll keep this in mind and always push patch updates.
